Dockerfile

```diff
@@ -3,17 +3,17 @@ LABEL maintainer="Charlie Lewis <clewis@iqt.org>"
 
 COPY requirements.txt requirements.txt
 
-RUN apt-get update && apt-get install -y --no-install-recommends \
-    curl \
-    gcc \
-    git \
-    python3 \
-    python3-dev \
-    tcpdump \
+ENV DEBIAN_FRONTEND=noninteractive
+ENV BUILD_DEPS="curl gcc git python3.6-dev"
+
+RUN apt-get update && apt-get install -yq --no-install-recommends \
+    $BUILD_DEPS \
+    python3.6 \
+    tshark \
```

Comment:
The current PR fails one of the Travis tests and the unit tests fail too. Ryan Ashley and I were able to make this PR run either by adding in a new directory to hold a Java development kit man page (which is strange) or changing the Python version to Python 3.7. There seems to be a dependency on Java, which is unexpected. Can you please investigate?

Reply:
Sorry about that. Yes it appears python3.6 specifically is bringing in an unexpected dependency on X11 and Java which is bad. This appears to be a recent (last few days) dependency change. I fixed it by avoiding the need to bring in 3.6 entirely and using Travis to test 3.6 (which it didn't before).

Reply:
Thanks Josh.

Reply:
This is confirmed fixed everything is passing.